.drone.yml

```diff
@@ -55,9 +55,10 @@ trigger:
   - pull_request
   paths:
     exclude:
-    - '*.md'
     - docs/**
-    - latest.json
+    - pkg/**
```

Comment:
Should probably exclude go.mod/sum changes. Exclude latest.json and all *.md files?